pull-requests.md

```diff
@@ -0,0 +1,164 @@
+- [**Open source etiquette**](https://developer.mozilla.org/en-US/docs/MDN/Contribute/Open_source_etiquette)
+
+## Contributing to MDN
+
+This section describes how contributors make changes on MDN Web Docs and how the changes are reviewed and land on the site.
+Content changes to MDN Web Docs are related to a variety of work streams, including:
+
+- **Day-to-day content improvements:** documentation of new APIs, CSS properties, platform updates and content additions.
+  This is usually done by MDN Web Docs staff working for Mozilla, Google, Open Web Docs, Samsung, but also by community volunteers.
+- **Minor fixes:** small updates done to the site to fix typos, grammatical issues, and technical inaccuracies.
+  These issues are usually found by users of MDN Web Docs.
+- **Content bug fixes:** usually done by volunteers to close issues on `mdn/content` repo.
+
+Regardless of how content changes are done, they are submitted as pull requests on GitHub. The content changes go through the following stages before they are published on MDN Web Docs:
+
+1. **Submission:** As a pull request author, you submit the changes. There are a few guidelines you need to be aware of before and after you open a pull request.
+2. **Review:** Your changes are reviewed by MDN staff members and volunteers.
+   When a pull request related to a particular topic is opened, it will be assigned to that area's topic review owners.
+   The `CODEOWNERS` file specifies the teams who own certain content directories.
+3. **Approval:** The changes can be merged after a approving review has been made.
+   The assigned reviewer will merge the pull request.
+4. **Publishing:** Updated content goes live within a day of merging via a site rebuild once every 24 hours.
+
+### Decision tree
+
+TODO: example decision tree in mermaid
+
+```mermaid
+graph TB
+  A("Do you think<br> online learning is right for you?")
+  B("Do you have time to design<br> a service learning component?")
+  C("What is the civic or public purpose of your discipline?<br> How do you teach that without service learning?")
+  D("Do you have departmental or school<br> support to plan and implement service learning?")
+  E["Are you willing to be a trailblazer?"]
+  F["What type of service learning to you want to plan?"]
+
+  A==Yes==>B
+  A--No-->C
+  B==Yes==>D
+  B--No-->C
+  D--Yes-->F
+  D--No-->E
+  E--Yes-->F
+  E--No-->C
+```
+
+## Guidelines for opening a pull request
+
+### Before you open a pull request
+
+Anyone who plans to open a PR to make a change on MDN Web Docs should:
+
+- **Open a discussion** to describe why you think the changes are required to improve the content.
+  This is important if your PR is a technical change, not typo fixes, grammatical improvements, formatting or structural changes.
+- **File an issue describing what the PR will change.**
+  When you submit a PR, reference this issue using `#` followed by the issue's ID, for example `#1234`.
```

Comment:
I think we can split this into:

* Opening a PR because you want to make a change
* Opening a PR to fix an existing issue (for instance from the contributors project board)

We should add the content proposal process here. Also folks are directed to raising a new issue from MDN directly - ok ok I guess this is a case for a flow chart :)

Reply:
😄 I think this is more clear here after https://github.com/mdn/temp-processes/pull/1/commits/ea80118c479ccb45cb3d20a5adac49d143ce6bdf